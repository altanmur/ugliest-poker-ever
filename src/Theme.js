import React from 'react';
import PropTypes from 'prop-types';
import { ThemeProvider } from 'styled-components';

export const theme = {
  colors: {
    indigo: '#4A0968',
    blackcurrant: '#27172F',
  },
};

const Theme = ({ children }) => (
  <ThemeProvider theme={theme}>{children}</ThemeProvider>
);

Theme.propTypes = {
  children: PropTypes.element.isRequired,
};

export default Theme;
