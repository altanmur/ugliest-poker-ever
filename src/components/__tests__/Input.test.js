import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Input from '../Input';
import { theme } from '../../Theme';

describe('Input Component', () => {
  it('should render correctly', () => {
    const component = shallow(<Input theme={theme} />);

    expect(component).toMatchSnapshot();
  });
});
