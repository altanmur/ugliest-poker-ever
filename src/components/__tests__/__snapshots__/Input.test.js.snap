// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component should render correctly 1`] = `
.c0 {
  background: #3ea300;
  border: none;
  color: #1b2f04;
  width: 100%;
  text-align: center;
  padding: 4px;
  margin: 8px 0;
  outline: none;
  font-weight: bold;
}

.c0::-webkit-input-placeholder {
  color: #1b2f04;
  font-weight: bold;
}

.c0::-moz-placeholder {
  color: #1b2f04;
  font-weight: bold;
}

.c0:-ms-input-placeholder {
  color: #1b2f04;
  font-weight: bold;
}

.c0::placeholder {
  color: #1b2f04;
  font-weight: bold;
}

<input
  className="c0"
/>
`;
